bayes_optim/Solution.py

```diff
@@ -6,6 +6,7 @@
 
 """
 import numpy as np
+import pandas as pd
```

Comment:
I was always trying to stay away from pandas, since it is a bi slow and I do not like it..
anyway, we could support it here. What is the expected use case of the data frames?